css/style.css

```diff
@@ -1,4 +1,384 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  box-sizing: border-box; 
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+  padding: 21px 5px 74px 0px;
+}
+
+.header__logo-img {
+  width: 163px;
+  height: 48px;
+}
+.menubar {
+  display: flex;
+  align-items: center;
+}
+
+.menubar__link {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, 
+.menubar__link:active, 
+.menubar__link:focus {
+  cursor: pointer;
+  color: #e51937;
+  box-shadow: 0 2px 0 #e51937;
+  opacity: 0.75;
+  transition: 0.4s;
+}
+
+.menubar__link_red:nth-child(1) {
+  margin-right: 63px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-right: 25px;
+}
+
+.menubar__link:nth-child(3) {
+  margin-right: 44px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-right: 48px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-right: 221px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.button {
+  display: block;
+  background-color: #fff;
+  width: 24px;
+  height: 16px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+}
+
+.button:nth-child(8) {
+  margin-right: 10px;
+}
+
+.button__line {
+  position: absolute;
+  width: inherit;
+  height: 2px;
+  background-color: #e51937;
+}
+
+.second {
+  top: 7px;
+}
+
+.third {
+  top: 14px;
+}
+
+.button:hover, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)